roles/convert2rhel/tasks/host_groups.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: "Create 'CentOS 7 converting' host group"
+  theforeman.foreman.hostgroup:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    name: "{{ foreman_c2r_hostgroup7 }}"
+    lifecycle_environment: "{{ foreman_c2r_lifecycle_env }}"
+    content_view: "{{ foreman_c2r_content_view }}"
+    parameters:
+      - name: "kt_activation_keys"
+        value: "{{ foreman_c2r_ack_centos7 }}"
```

Comment:
not strictly required, but you could use `activation_keys: "{{ foreman_c2r_ack_centos7 }}"` instead, which will set the right parameter automatically.